### PR TITLE
Add ADTV to the marketdata examples

### PR DESCRIPTION
**Pull request by gnvk**

### CODE REVIEW

File: examples/marketdata/marketdata.go [hunk 1]
```diff
@@ -50,4 +50,35 @@ func main() {
...
+func getADTV(symbol string, start, end time.Time) (av float64, n int, err error) {
+	var (
+		totalVolume uint64
+	)
+	for item := range marketdata.GetBarsAsync(symbol, marketdata.GetBarsParams{
+		Start: start,
+		End:   end,
+	}) {
+		if err = item.Error; err != nil {
+			return
```
Comment (kalmant) [inline examples/marketdata/marketdata.go:74]: I think a panic or printing an error or something would be prudent here

Comment (gnvk) [reply examples/marketdata/marketdata.go:74]: We return the error and the caller will panic.

Comment (kalmant) [reply examples/marketdata/marketdata.go:74]: You are right, my bad

**Review by kalmant — APPROVED** [review]
Thank you for adding this example.

**Pull request merged**